src/client/src/apps/MainRoute/widgets/spotTile/components/PriceControls/PriceControls.tsx

```diff
@@ -49,69 +49,67 @@ const PriceControls: React.FC<Props> = ({
 
   const isDisabled = !isRfqStateReceived && disabled
   const hasPrice = Boolean(priceData.bid && priceData.ask)
+  let priceButtonDisabledStatus: JSX.Element = null
+  let priceButtonBuy: JSX.Element = null
+  let priceButtonSell: JSX.Element = null
+
+  if (isRfqStateNone || isRfqStateReceived || isRfqStateExpired || isTradeExecutionInFlight) {
```

Comment:
Might be worth putting these 4 conditions  in a small function

Reply:
I updated this file along with the TileBusinessLogic to clean things up a little